augur/data/schema-multiplot.json

```diff
@@ -0,0 +1,126 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://nextstrain.org/schemas/multiplot/v1",
+    "title": "Nextstrain multiplot data schema proposal",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["config", "collections"],
+    "properties": {
+        "config": {
+            "description": "Config for display text and display options",
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["collections", "groupings"],
+            "properties": {
+                "collections": {
+                    "description": "The available collection options and their display titles",
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "description": "A single collection option",
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": ["key"],
+                        "properties": {
```

Comment:
Additional properties to consider:
* `plot-title` - alternatively we could reuse the `title` instead of separating them. (Relatedly, the frequency plot is an example where we change the plot title depending on the state.)
* `threshold` (see below). 
* `x-axis label`
* `description` - e.g. a paragraph detailing the assay behind the data